include/leo_http.hrl

```diff
@@ -390,6 +392,7 @@
           custom_header_settings     :: list() | undefined,     %% http custom header settings
           timeout_for_header         :: pos_integer(),          %% Timeout for reading header
           timeout_for_body           :: pos_integer(),          %% Timeout for reading body
+          sending_chunk_len = 0      :: pos_integer(),          %% sending chunk length
```

Comment:
This new parameter is the send version corresponding to the existing parameter 'reading_chunked_obj_len' the read version.
So it'd be good to comply with the existing naming rule for LeoFS users.

s/sending_chunk_len/sending_chunked_obj_len/